modules/components/core/org.wso2.carbon.pc.core/src/main/java/org/wso2/carbon/pc/core/ProcessStore.java

```diff
@@ -1248,8 +1250,11 @@ public String uploadDocument(String processName, String processVersion, String d
             }
         }
         try {
-            StreamHostObject s = (StreamHostObject) docObject;
-            InputStream docStream = s.getStream();
+            if(docUrl.equals("NA")) {
```

Comment:
This code has been moved to ProcessDocument.java. Fixed in there with https://github.com/wso2/product-pc/commit/e931bb704e9065cf57f08f61619a45a4beecb5c7